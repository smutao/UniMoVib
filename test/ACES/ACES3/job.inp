a test job of ACES 3.

 $contrl
   qcprog="aces"
 $end

 $qcdata
   fchk="ch4-a3.out"
 $end
