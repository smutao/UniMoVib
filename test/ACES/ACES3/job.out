 =======================================================================================================================
       _/     _/                _/     _/            _/     _/
      _/     _/            _/  _/|  _/_/            _/     _/  _/  _/        UniMoVib:          A unified interface for
     _/     _/   _/_/_/       _/ |_/ _/   _/_/_/   _/     _/      _/           molecular harmonic vibrational frequency
    _/     _/  _/    _/  _/  _/  _/ _/  _/    _/  _/     _/  _/  _/_/_/        calculations.
   _/     _/  _/    _/  _/  _/     _/  _/    _/  _/    _/   _/  _/   _/
  _/     _/  _/    _/  _/  _/     _/  _/    _/   _/  _/    _/  _/   _/       Ver. 1.1.1, Jan 14, 2018
   _/_/_/   _/    _/  _/  _/     _/   _/_/_/     _/_/     _/  _/_/_/         https://github.com/zorkzou/UniMoVib
 =======================================================================================================================


 QC Program:      ACES
 FCHK file:       ch4-a3.out


 Cartesian coordinates (Angstrom)
 ------------------------------------------------------------------------------------------
   No.   Atom    ZA                 X             Y             Z                Mass
 ------------------------------------------------------------------------------------------
     1    H       1            0.00000000    0.88418541   -0.62521350            1.00782504
     2    C       6            0.00000000    0.00000000    0.00000000           12.00000000
     3    H       1            0.00000000   -0.88418541   -0.62521350            1.00782504
     4    H       1            0.88418541    0.00000000    0.62521350            1.00782504
     5    H       1           -0.88418541    0.00000000    0.62521350            1.00782504
 ------------------------------------------------------------------------------------------


 <<< SYMMETRY >>>
 Molecular Point Group (Z)   :   Td  
 Molecular Point Group (Z+M) :   Td  


 ************************************
 ***  Properties of Normal Modes  ***
 ************************************

 Results of vibrations:
 Normal frequencies (cm**-1), reduced masses (AMU), force constants (mDyn/A)

                                                   1                                 2                                 3
          Irreps                                  T2                                T2                                T2
     Frequencies                           1520.3098                         1520.3298                         1520.3483
  Reduced masses                              1.1821                            1.1821                            1.1821
 Force constants                              1.6099                            1.6099                            1.6099
        Atom  ZA               X         Y         Z             X         Y         Z             X         Y         Z
           1   1        -0.57063   0.10239   0.11338       0.19131   0.23052   0.20803       0.05713   0.25096   0.43578
           2   6         0.11886   0.00384  -0.04142      -0.03985   0.04638  -0.11009      -0.01190  -0.11701  -0.04499
           3   1        -0.57063  -0.11124   0.13316       0.19131  -0.33742   0.44736       0.05713   0.01878  -0.16790
           4   1        -0.24383  -0.01841   0.42990      -0.23800  -0.22268   0.22490      -0.10235   0.56174   0.10323
           5   1        -0.03021  -0.01841  -0.18331       0.32990  -0.22268   0.43048       0.12979   0.56174   0.16462

                                                   4                                 5                                 6
          Irreps                                   E                                 E                                A1
     Frequencies                           1739.7779                         1739.7801                         3186.7586
  Reduced masses                              1.0078                            1.0078                            1.0078
 Force constants                              1.7973                            1.7973                            6.0302
        Atom  ZA               X         Y         Z             X         Y         Z             X         Y         Z
           1   1        -0.50000   0.00000   0.00000      -0.00001   0.28871   0.40821      -0.00001  -0.40822   0.28872
           2   6         0.00000   0.00000   0.00000       0.00000   0.00000  -0.00000       0.00000  -0.00000  -0.00000
           3   1         0.50000  -0.00000   0.00000      -0.00001  -0.28869   0.40824      -0.00001   0.40822   0.28871
           4   1         0.00000  -0.50000  -0.00000       0.28871  -0.00001  -0.40823      -0.40824   0.00000  -0.28870
           5   1        -0.00000   0.50000  -0.00000      -0.28873  -0.00001  -0.40822       0.40822   0.00000  -0.28870

                                                   7                                 8                                 9
          Irreps                                  T2                                T2                                T2
     Frequencies                           3280.1027                         3280.1134                         3280.1321
  Reduced masses                              1.0989                            1.0989                            1.0989
 Force constants                              6.9661                            6.9661                            6.9661
        Atom  ZA               X         Y         Z             X         Y         Z             X         Y         Z
           1   1         0.01880   0.44893  -0.31694      -0.00749   0.26451  -0.21297       0.01368  -0.47303   0.31957
           2   6         0.07010  -0.04651   0.03477      -0.02791  -0.07479  -0.04375       0.05092   0.02303  -0.07185
           3   1         0.01880   0.12977   0.10990      -0.00749   0.66610   0.47341       0.01368   0.18645   0.10818
           4   1        -0.59573  -0.01248  -0.42520       0.37443  -0.02004   0.25832       0.01292   0.00619  -0.01980
           5   1        -0.27656  -0.01248   0.21820      -0.02713  -0.02004   0.00215      -0.64655   0.00619   0.44759

 Results of translations and rotations:

                                                   1                                 2                                 3
          Irreps                                  T2                                T2                                T2
     Frequencies                              8.7627                           -3.4147                           -6.7234
  Reduced masses                              3.2063                            3.2063                            3.2063
 Force constants                              0.0001                           -0.0000                           -0.0001
        Atom  ZA               X         Y         Z             X         Y         Z             X         Y         Z
           1   1         0.44721   0.00000   0.00000       0.00000   0.44721   0.00000       0.00000   0.00000   0.44721
           2   6         0.44721   0.00000   0.00000       0.00000   0.44721   0.00000       0.00000   0.00000   0.44721
           3   1         0.44721   0.00000   0.00000       0.00000   0.44721   0.00000       0.00000   0.00000   0.44721
           4   1         0.44721   0.00000   0.00000       0.00000   0.44721   0.00000       0.00000   0.00000   0.44721
           5   1         0.44721   0.00000   0.00000       0.00000   0.44721   0.00000       0.00000   0.00000   0.44721

                                                   4                                 5                                 6
          Irreps                                  T1                                T1                                T1
     Frequencies                              0.0000                            0.2278                            0.2237
  Reduced masses                              1.0078                            1.0078                            1.0078
 Force constants                              0.0000                            0.0000                            0.0000
        Atom  ZA               X         Y         Z             X         Y         Z             X         Y         Z
           1   1         0.50000   0.00000   0.00000      -0.00000  -0.35355  -0.50000      -0.35355  -0.00000  -0.00000
           2   6         0.00000   0.00000   0.00000      -0.00000   0.00000  -0.00000       0.00000   0.00000  -0.00000
           3   1        -0.50000   0.00000   0.00000       0.00000  -0.35355   0.50000      -0.35355  -0.00000   0.00000
           4   1         0.00000  -0.50000  -0.00000      -0.00000   0.35355   0.00000       0.35355   0.00000  -0.50000
           5   1         0.00000   0.50000   0.00000      -0.00000   0.35355   0.00000       0.35355  -0.00000   0.50000


 *********************************************
 ***   Thermal Contributions to Energies   ***
 *********************************************

 Molecular mass            :        16.031300    AMU
 Elec. total energy        :         0.000000    Hartree
 Scale factor of Frequency :         1.000000
 Rot. symmetry number      :        12
 The Td   point group is used to calculate rotational entropy.

 Principal axes and moments of inertia in atomic units:
                                     1                   2                   3
     Eigenvalues --                 11.254587           11.254587           11.254587
           X                         0.000000            1.000000            0.000000
           Y                        -0.000000            0.000000           -1.000000
           Z                         1.000000            0.000000           -0.000000

 Rotational temperatures             7.695868            7.695868            7.695868    Kelvin
 Rot. constants A, B, C              5.348902            5.348902            5.348902    cm**-1
                                   160.356059          160.356058          160.356058    GHz


 #   1    Temperature =       298.15000 Kelvin         Pressure =         1.00000 Atm
 ====================================================================================

 Thermal correction energies                              Hartree            kcal/mol
 Zero-point Energy                          :            0.047996           30.117721
 Thermal correction to Energy               :            0.050845           31.905929
 Thermal correction to Enthalpy             :            0.051790           32.498415
 Thermal correction to Gibbs Free Energy    :            0.030702           19.265651
 ====================================================================================


 UniMoVib job terminated correctly! Sun Jan 14 13:17:49 2018
